src/github.rs

```diff
@@ -208,49 +210,69 @@ pub async fn command_fetch_release_distributions(args: &ArgMatches) -> Result<()
 
             let name = zf.name().to_string();
 
+            let parts = name.split('-').collect::<Vec<_>>();
+
+            if parts[0] != "cpython" {
+                println!("ignoring {} not a cpython artifact", name);
+                continue;
+            };
+
+            let python_version = pep440_rs::Version::from_str(parts[1])?;
+            if !release_version_range.contains(&python_version) {
+                println!(
+                    "{} not in release version range {}",
+                    name, release_version_range
+                );
+                continue;
+            }
+
             // Iterate over `RELEASE_TRIPLES` in reverse-order to ensure that if any triple is a
             // substring of another, the longest match is used.
-            if let Some((triple, release)) =
+            let Some((triple, release)) =
```

Comment:
Best viewed with [unified whitespace](https://github.com/indygreg/python-build-standalone/pull/340/files?diff=unified&w=1) — this section is dedented.